README.md

```diff
@@ -1,2 +1,48 @@
-# video-store-cli
-A Command Line Interface for Retro Video Store API
+# Video Store CLI
+
+Once upon a time in ye olden days to watch a movie a person had to head down to their local video store and rent a video.  We are going to step into the shoes of this old timey retro video store owner and build for ourselves all of the tools that we need in order to run our successful corner video store. 
+
+You have already built a [Video Store API](https://github.com/AdaGold/retro-video-store) for the backend in Part 1 of the project.
+
+In keeping with our retro vibe, for this part of the project we will build a Command Line Interface (CLI) for the front end.
+
+## Manual Tests
+
+We will be testing your CLI manually to see how it performs all of the required behaviors.  We recommend that you find a partner to "play-test" your CLI and put it through it's paces. 
```

Comment:
I'm a little worried about how far we're going in the direction of ambiguity here. Namely, how that might create unhelpful anxiety for students and how that might cause Instructors to get a lot of repeated questions about what is and isn't expected.
I think if we want it to be this open-ended, I might prefer we just say something like "We're not going to review this very strictly, have fun with it! Use your creativity!"

Reply:
Yeah, either tell them to go wild, or give them some user stories to complete.

Reply:
I mean, this does have clear user stories to complete. My concern is just that there's no further explanation about how to handle edge cases within user stories and what edge cases to worry about